draft-ietf-webpush-encryption.md

```diff
@@ -416,7 +421,7 @@ Shared ECDH secret (ecdh_secret):
 
 : kyrL1jIIOHEzg3sM2ZWRHDRB62YACZhhSlknJ672kSs
 
-Pseudo-random key for key combining (PRK_key):
+Pseudorandom key (PRK) for key combining (PRK_key):
```

Comment:
Note that this is still a backwards reference, as "PRK" appears earlier in the document.